front/src/applications/editor/tools/rangeEdition/speedSection/SpeedSectionMetadataForm.tsx

```diff
@@ -31,6 +32,17 @@ const SpeedSectionMetadataForm: FC = () => {
     setState,
   } = useContext(EditorContext) as ExtendedEditorContextType<RangeEditionState<SpeedSectionEntity>>;
 
+  const [uniqueIds, setUniqueIds] = useState<string[]>([]);
+
+  useEffect(() => {
+    if (!isEmpty(entity.properties.speed_limit_by_tag)) {
+      const newUniqIds: string[] = Object.keys(entity.properties.speed_limit_by_tag).map(() =>
+        nextId()
+      );
+      setUniqueIds(newUniqIds);
+    }
+  }, []);
```

Comment:
If we try to edit a speed section who already have some speed limit by tags, we still get at first load the "duplicated key" warning in console. 
Maybe even with the useEffect with an empty dependency array, the first render of the components starts before the state has been updated.

It seems that the solution below fixes the warning.

```suggestion
  const [uniqueIds, setUniqueIds] = useState<string[]>(
    Object.keys(entity.properties?.speed_limit_by_tag || {}).map(() => nextId())
  );
```

Reply:
Done !